cli/alsatian-cli-options.ts

```diff
@@ -73,7 +73,7 @@ export class AlsatianCliOptions {
          const argumentIndex = this._getArgumentIndexFromArgumentList(args, "timeout", "t");
 
          // filter out the timeout argument and its value
-         return args.filter((value, index) => {
+         return args.filter((_, index) => {
```

Comment:
This is one thing I don't like about the no unused arguments. Granted `filter` is a common function but it's obscuring argument names for no good reason.